test/testcafe/spec/ChallengeAuthenticatorEmail_spec.js

```diff
@@ -274,3 +284,25 @@ test
     await t.expect(await terminalPageObject.goBackLinkExists()).ok();
     await t.expect(terminalPageObject.getFormTitle()).eql('Verify with your email');
   });
+
+test
+  .requestHooks(dynamicRefreshIntervalMock)('challenge email authenticator screen has right labels', async t => {
+    await setup(t);
+
+    await t.wait(31000);
```

Comment:
this is taking total more than a minute to finish.. lets just validates dynamic interval.. with shorter wait time say `t.wait(5000)` and refreshInterval even shorter say `1000`

Reply:
fixed to total of 8 sec (2 sec with 1 sec/req interval + 6 sec with 2 sec/req interval)